generators/javascript/javascript_generator.ts

```diff
@@ -126,8 +126,8 @@ export class JavascriptGenerator extends CodeGenerator {
     // this list is trivial.  This is intended to prevent users from
     // accidentally clobbering a built-in object or function.
     this.addReservedWords(
-        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords
-        'break,case,catch,class,const,continue,debugger,default,delete,do,' +
+      // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords
+      'break,case,catch,class,const,continue,debugger,default,delete,do,' +
         'else,export,extends,finally,for,function,if,import,in,instanceof,' +
```

Comment:
This is a weird thing it did. Might want to report this as a prettier bug. Or just move the comment to outside the parameter list (above `this.addReservedWords` instead).

Reply:
I moved the comment since I thought that worth doing anyway, but it did not result in prettier formatting the argument any differently, and indeed I think it is actually doing the right thing here: there is a single argument to `addReservedWords`, and that argument is in fact split over multiple lines, and the normal rule is that continuation lines should be indented (+4 in Google style, +2 in Prettier style).